src/styles/CompareCards.ts

```diff
@@ -23,6 +23,16 @@ export const CompareCardsStyles = (mode: string) => {
   const isTrueLight = mode == 'light' ? true : false;
 
   const compareCardsCSS = stylesheet({
+    wrapper: {
+      $nest: {
+        '&.wrapper--withbase': {
+          marginBottom: `${Spacing.Large}px`,
+        },
+        '&.wrapper--overtime': {
+          marginBottom: `${Spacing.layoutXLarge - 12}px`,
+        },
+      },
+    },
```

Comment:
I don't understand why we're doing overrides like this, instead of defining them directly in the component itself.
I only see inconvenients and no advantages.

Inconvenients:
* it's necessary to look in 2 different files to find the overrides.
* from CompareOverTime or CompareWithBase, it's not obvious that overrides exist at all.
* we need to introduce CSS classes that are used just for this. Later on if we remove these styles, the classes will probably stay on the component for no good reason. (that wouldn't be a big deal though).

Advantages:
* the advantage of adding things here is usually (IMO) when we _share_ styles. But that's not the case here as these styles are overrides, they're not shared. So this advantage doesn't apply for these styles.

I know that a lot of other styles are similar, but I think this should be changed eventually. Let's try to not add more this way... But please tell me what you think, especially if I'm missing something!

Additional question: why do you have this `-12` for compare over time? Can we use another constant somewhere? Or at the very least please add a comment about it to explain the reason.

Reply:
Yes, it makes sense to keep unique styles within the components. I don't mind doing that from now on. No prob. I'll make a spacing constant for 12. Not sure why it didn't exist before.